src/script4.R

```diff
@@ -0,0 +1,114 @@
+# authors: Fatime Selimi, Nicholas Wu, Tanmay Sharma, Neel Phaterpekar
+# date: 2019-12-28
+
+"Performs statistical analysis on preprocessed data and save figures and tables 
+into new file.
+
+Usage: script4.R --data=<data> --sum_data=<sum_data> --out_dir=<out_dir>
+ 
+Options:
+--data=<data>          Path to the file containing the processed data (in standard csv format)
+--sum_data=<sum_data>   Path to the file containing the summary data (in standard csv format)
+--out_dir=<out_dir>   Path (including filename) of where to locally save the figures and tables
+"-> doc
+
+library(docopt)
+library(readr)
+library(tidyverse)
+library(kableExtra)
+library(broom)
+library(infer)
+
+opt <- docopt(doc)
+
+# main function to call 
+
+main <- function(data, sum_data, out_dir) {
+  data <- read_csv(data)
+  sum_data <- read_csv(sum_data)
```

Comment:
Same here. We should remove the `sum_data` parameter when we call the main function, as well as this line of code `sum_data <- read_csv(sum_data)` since `sum_data` is not being used anywhere in the code.

Reply:
I use sum_data to calculate the t-stat.

Reply:
Should I change script #3 so it saves a .csv file for the summary statistic for script #4 to work? Currently I'm only saving it as a .png

Reply:
@nichowu yes, please. We've merged your PR to the main branch so you should create a new PR for that change.

Reply:
Updated. See #40